File: OAKFITNESSjava/src/main/java/com/mycompany/oakfitnessjava/entities/Exercice.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.oakfitnessjava.entities;

/**
 *
 * @author Heni Nechi
 */
public class Exercice {

    private int IDExercice;
    private String TypeExercice;
    private String NomExercice;
    private String Muscle;
    private String Video;
    private String DescrExercice;
    private String DiffExercice;
    private String JusteSalleExercice;
    private String DureeExercice;

    public Exercice() {

    }

    public Exercice(int IDExercice, String TypeExercice, String NomExercice,String Muscle,String Video, String DescrExercice, String DiffExercice, String JusteSalleExercice, String DureeExercice) {
        this.IDExercice = IDExercice;
        this.TypeExercice = TypeExercice;
        this.NomExercice = NomExercice;
        this.Muscle = Muscle;
        this.Video = Video;
        this.DescrExercice = DescrExercice;
        this.DiffExercice = DiffExercice;
        this.JusteSalleExercice = JusteSalleExercice;
        this.DureeExercice = DureeExercice;
    }

    public Exercice(String TypeExercice, String NomExercice,String Muscle,String Video, String DescrExercice, String DiffExercice, String JusteSalleExercice, String DureeExercice) {
        this.TypeExercice = TypeExercice;
        this.NomExercice = NomExercice;
        this.Muscle = Muscle;
        this.Video = Video;
        this.DescrExercice = DescrExercice;
        this.DiffExercice = DiffExercice;
        this.JusteSalleExercice = JusteSalleExercice;
        this.DureeExercice = DureeExercice;
    }

    public String getMuscle() {
        return Muscle;
    }

    public void setMuscle(String Muscle) {
        this.Muscle = Muscle;
    }

    public String getVideo() {
        return Video;
    }

    public void setVideo(String Video) {
        this.Video = Video;
    }

    public int getIDExercice() {
        return IDExercice;
    }

    public void setIDExercice(int IDExercice) {
        this.IDExercice = IDExercice;
    }

    public String getTypeExercice() {
        return TypeExercice;
    }

    public void setTypeExercice(String TypeExercice) {
        this.TypeExercice = TypeExercice;
    }

    public String getNomExercice() {
        return NomExercice;
    }

    public void setNomExercice(String NomExercice) {
        this.NomExercice = NomExercice;
    }

    public String getDescrExercice() {
        return DescrExercice;
    }

    public void setDescrExercice(String DescrExercice) {
        this.DescrExercice = DescrExercice;
    }

    public String getDiffExercice() {
        return DiffExercice;
    }

    public void setDiffExercice(String DiffExercice) {
        this.DiffExercice = DiffExercice;
    }

    public String getJusteSalleExercice() {
        return JusteSalleExercice;
    }

    public void setJusteSalleExercice(String JusteSalleExercice) {
        this.JusteSalleExercice = JusteSalleExercice;
    }

    public String getDureeExercice() {
        return DureeExercice;
    }

    public void setDureeExercice(String DureeExercice) {
        this.DureeExercice = DureeExercice;
    }

    @Override
    public String toString() {
        return "Exercice : " + NomExercice + " | Muscle : " + Muscle + " | Exercice Duration : " + DureeExercice + "";
    }

   

}

File: src/Entity/Exercice.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Exercice
 *
 * @ORM\Table(name="exercice")
 * @ORM\Entity
 */
class Exercice
{
    /**
     * @var int
     *
     * @ORM\Column(name="IDExercice", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="TypeExercice", type="string", length=20, nullable=false)
     */
    private $typeexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="NomExercice", type="string", length=50, nullable=false)
     */
    private $nomexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="Muscle", type="string", length=70, nullable=false)
     */
    private $muscle;

    /**
     * @var string
     *
     * @ORM\Column(name="Video", type="string", length=150, nullable=false)
     */
    private $video;

    /**
     * @var string
     *
     * @ORM\Column(name="DescrExercice", type="string", length=2000, nullable=false)
     */
    private $descrexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="DiffExercice", type="string", length=20, nullable=false)
     */
    private $diffexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="JusteSalleExercice", type="string", length=20, nullable=false)
     */
    private $justesalleexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="DureeExercice", type="string", length=30, nullable=false)
     */
    private $dureeexercice;

    /**
     * @var string
     *
     * @ORM\Column(name="Image", type="string", length=200, nullable=false)
     */
    private $image;


}
